src/senaite/core/browser/static/js/senaite.core.analysisrequest.add.js

```diff
@@ -1,5 +1,5 @@
   /* Please use this command to compile this file into the parent `js` directory:
-      coffee --no-header -w -o -b ../ -c bika.lims.analysisrequest.add.coffee
+      coffee --no-header -w -o -b -c senaite.core.analysisrequest.add.coffee
```

Comment:
Would you mind to correct this line in the `senaite.core.analysisrequest.add.coffee` to this:

`coffee --no-header -w -o . -b -c senaite.core.analysisrequest.add.coffee`

I missed the to set the output directory (`-o`) correctly the last tim

Reply:
or maybe just remove the `-o` completely 🤔 The same directory should be the default, right?